gazebo/11/ubuntu/focal/gzserver11/Dockerfile

```diff
@@ -13,15 +13,22 @@ RUN echo 'Etc/UTC' > /etc/timezone && \
 RUN apt-get update && apt-get install -q -y --no-install-recommends \
     dirmngr \
     gnupg2 \
-    lsb-release \
     && rm -rf /var/lib/apt/lists/*
 
+
 # setup keys
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
+RUN set -eux; \
+	key='D2486D2DD83DB69272AFE98867170598AF249743'; \
+	export GNUPGHOME="$(mktemp -d)"; \
+	gpg --batch --keyserver keyserver.ubuntu.com --recv-keys "$key"; \
+	mkdir -p /usr/share/keyrings; \
+	gpg --batch --export "$key" > /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg; \
```

Comment:
@j-rivero I followed the install location from [the gazebo docs](https://gazebosim.org/docs/harmonic/install_ubuntu)

Do you confirm the keyring should be installed in `/usr/share/keyrings` and not in `/etc/apt/keyrings`?
Same for the ROS ones ? (@nuclearsandwich ?)

Reply:
> Do you confirm the keyring should be installed in `/usr/share/keyrings` and not in `/etc/apt/keyrings`?
> Same for the ROS ones ? (@nuclearsandwich ?)

Following https://wiki.debian.org/DebianRepository/UseThirdParty
> If future updates to the certificate will be managed by an apt/dpkg package as recommended below, then it SHOULD be downloaded into /usr/share/keyrings using the same filename that will be provided by the package. If it will be managed locally , it SHOULD be downloaded into /etc/apt/keyrings instead.

Our intention is to manage the ROS/Gazebo keys using a package )hopefully sometime in the short future). So I'm inclined to use `/usr/share/keyrings` as suggested by the Debian documentation.